uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-comments-tab.component.spec.js

```diff
@@ -85,10 +86,10 @@ define([
 
       it('adds comment with proper values', function () {
         expect(controller.request.comments[0]).toEqual({
-          contact_id: staffId,
+          contact_id: contactId,
           created_at: jasmine.any(String),
```

Comment:
I removed the line, but after some short investigation I don't think it is actually needed to test if it is *not* passed. It makes sense to test if something is passed, but I am not quite sure testing opposite is necessary in this case.